pkg/controller/v1beta1/inferenceservice/reconcilers/service/service_reconciler.go

```diff
@@ -98,12 +130,16 @@ func createService(componentMeta metav1.ObjectMeta, componentExt *v1beta1.Compon
 			}
 		} else {
 			port, _ := strconv.Atoi(constants.InferenceServiceDefaultHttpPort)
+			portInt32, err := utils.ConvertStringToInt32(port, 8080)
+			if err != nil {
+				log.Error(err, "Failed to convert port int to int32, using the default value(8080) for port.")
```

Comment:
Suppose an error happened here, and we will still proceed by setting an invalid value?